index.bs

```diff
@@ -668,7 +668,8 @@ To <dfn>process the lock request queue</dfn> |queue|:
         1. [=set/Append=] |lock| to |origin|'s [=held lock set=].
         1. [=Enqueue the following steps=] on |callback|'s [=relevant settings object=]'s [=responsible event loop=]:
             1. If |signal| is present, then run these steps:
-                1. If |signal|'s [=AbortSignal/aborted flag=] is set, then return.
```

Comment:
This latest diff change dropped the **return** here - I think you still want to exit these overall steps early in this case, don't you?

Reply:
😬😬😬😬 yes, will add it right now